packages/special-pages/pages/onboarding/app/messages.js

```diff
@@ -51,6 +47,15 @@ export class OnboardingMessages {
      * @returns {Promise<InitResponse>}
      */
     async init () {
+        if (this.injectName === "integration") {
+            return {
+                stepDefinitions: {
+                    systemSettings: {
+                        rows: ["dock", "import", "default-browser"]
+                    }
+                }
+            }
+        }
```

Comment:
This is an example of bringing a simple default value into the correct place in the application - it's the minimal needed in this example to allow local debugging 

This kind of situation was highlighted recently with other contributors - in short we need to ensure the connection points to the browser don't prevent local development